src/main/js/portal/main/models/State.ts

```diff
@@ -65,10 +65,24 @@ export interface User extends WhoAmI {
 }
 
 type KnownDataObject = AsyncResult<typeof fetchKnownDataObjects>['rows'][0]
-export type ExtendedDobjInfo = AsyncResult<typeof getExtendedDataObjInfo>[0]
+export type ExtendedDobjInfo = {
+	dobj: string
```

Comment:
A very minor thing, but this could be more informative if written as `dobj: UrlStr`